src/test/java/com/google/step/datamanager/DealTagManagerDatastoreTest.java

```diff
@@ -0,0 +1,105 @@
+package com.google.step.datamanager;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class DealTagManagerDatastoreTest {
+  private static final long TAG_ID_A = 1;
+  private static final long TAG_ID_B = 2;
+  private static final long TAG_ID_C = 3;
+  private static final long TAG_ID_D = 4;
+
+  private static final long DEAL_ID_A = 1;
+  private static final long DEAL_ID_B = 2;
+  private static final long DEAL_ID_C = 3;
+  private static final long DEAL_ID_D = 4;
```

Comment:
Maybe use the constants you're defining in #27?

Reply:
I did not merge that into this branch because I want to merge that single file into master first, then we can all use those constants.